### PR TITLE
notification html and moving code

### PR DESCRIPTION
**Pull request by austenhale**
Adding html to show notifications.
Moved notification related code to notification section of codebase to remain consistent.

### CODE REVIEW

**Review by jalennall — APPROVED** [review]

File: CRUD/admin.py [hunk 1]
```diff
@@ -2,5 +2,8 @@
 from .models import AdjunctFacultyMember
 
 
+
```
Comment (JustinRaver) [inline CRUD/admin.py:5]: Remove these newlines

File: CRUD/forms.py [hunk 2]
```diff
@@ -31,6 +31,7 @@ class ClassForm(forms.ModelForm):
     fields = 'class_name'
 
 
-class NotificationForm(forms.ModelForm):
-    model = Notification
-    fields = 'message', 'date'
+# class NotificationForm(forms.ModelForm):
+#     class Meta:
```
Comment (JustinRaver) [inline CRUD/forms.py:35]: Move all notifications functionality into Notifications and remove from here

File: CRUD/models.py [hunk 3]
```diff
@@ -75,7 +75,7 @@ class Classes(models.Model):
     adj_class = models.CharField(max_length=10)
 
 
-class Notification(models.Model):
-    message = models.TextField()
-    date = models.DateField(default=datetime.now)
-    isRead = models.BooleanField()
+# class Notification(models.Model):
```
Comment (JustinRaver) [inline CRUD/models.py:78]: Remove all Notifications from here to Notifications

File: CRUD/views.py [hunk 4]
```diff
@@ -4,6 +4,7 @@
 from django.http import JsonResponse
 from .models import AdjunctFacultyMember
 from .forms import AdjunctForm
+# from .forms import NotificationForm
```
Comment (JustinRaver) [inline CRUD/views.py:7]: Remove

**Review by bethanyhull — APPROVED** [review]

**Pull request merged**